stubs/solvers.py

```diff
@@ -262,25 +261,25 @@ def init_petsc_matnest(self):
             self.Jpetsc_nest = d.PETScNestMatrix(Jpetsc).mat()
             # self.Jpetsc_nest = self.d_to_p(d.PETScNestMatrix(Jpetsc))
         self.Jpetsc_nest.assemble()
-        print(f"Jpetsc_nest assembled, size = {self.Jpetsc_nest.size}")
+        logger.info(f"Jpetsc_nest assembled, size = {self.Jpetsc_nest.size}")
```

Comment:
I think all loggers here and below (except the final warning) can be set to debug, for consistency with those above